spec/ruby/optional/capi/hash_spec.rb

```diff
@@ -91,6 +91,14 @@
     end
   end
 
+  describe "rb_hash_clear" do
+    it "returns self that cleared keys and values" do
+      hsh = { key: 'value' }
```

Comment:
Could you use the hash rocket syntax here? We'll probably want to backport this to our 1.8 branch which IIRC doesn't support the new symbol syntax.